### PR TITLE
gha: ci: Avoid using env unless it's really needed

### PR DESCRIPTION
**Pull request by fidencio**
de83cd9de79226b15519e07871dcdee635d501c6 tried to solve an issue, but it clearly seems that I'm using env wrongly, as what ended up being passed as input was "$VAR", instead of the content of the VAR variable.

As we can simply avoid using those here, let's do it and save us a headache.

Fixes: #7247

### CODE REVIEW

**Review by cmaf — APPROVED** [review]

**Review by dborquez — APPROVED** [review]

File: .github/workflows/ci-nightly.yaml [hunk 1]
```diff
@@ -18,6 +15,6 @@ jobs:
     needs: set-fake-pr-number
     uses: ./.github/workflows/ci.yaml
     with:
-      commit-hash: $COMMIT_HASH
-      pr-number: $PR_NUMBER
-      tag: $PR_NUMBER-$COMMIT_HASH-nightly
+      commit-hash: ${GITHUB_REF}
+      pr-number: ${PR_NUMBER}
+      tag: ${PR_NUMBE}R-${GITHUB_REF}-nightly
```
Comment (GabyCT) [inline .github/workflows/ci-nightly.yaml:20]: @fidencio is ${PR_NUMBER} instead of ${PR_NUMBE}R

**Comment by dborquez** [comment]
@fidencio I've tagged w/ WIP to prevent the PR from being merged with the typo found by @GabyCT

**Pull request merged**